### ISSUE TITLE
Token is null -> Exception in thread "session-packet-receiver" java.nio.BufferUnderflowException

### ISSUE THREAD

**Issue by designandcopy** [librespot-org/librespot-java#158]
Running on Raspberry Pi 2 via spocon, fresh install.

```
Dec 07 10:23:41 raspberrypi java[1351]: 2019-12-07 10:23:41 INFO  Session:102 - Created new session! {deviceId: REDACTED, ap: guc3-accesspoint-a-76tv.ap.spotify.com/104.154.127.211}

Dec 07 10:23:41 raspberrypi java[1351]: 2019-12-07 10:23:41 INFO  ZeroconfServer:312 - Accepted new user from REDACTED. {deviceId: REDACTED}

Dec 07 10:23:42 raspberrypi java[1351]: 2019-12-07 10:23:42 INFO  Session:252 - Connected successfully!

Dec 07 10:23:42 raspberrypi java[1351]: 2019-12-07 10:23:42 TRACE Session:751 - Handled Ping {payload: 5DEB6F9E}

Dec 07 10:23:42 raspberrypi java[1351]: 2019-12-07 10:23:42 DEBUG TokenProvider:37 - Token expired or not suitable, requesting again. {scope: playlist-read, token: null}

Dec 07 10:23:42 raspberrypi java[1351]: 2019-12-07 10:23:42 INFO  Session:788 - Skipping SecretBlock

Dec 07 10:23:42 raspberrypi java[1351]: Exception in thread "session-packet-receiver" java.nio.BufferUnderflowException

Dec 07 10:23:42 raspberrypi java[1351]: 2019-12-07 10:23:42 TRACE MercuryClient:131 - Send Mercury request, seq: 1, uri: hm://keymaster/token/authenticated?scope=playlist-read&client_id=REDACTED&device_id=REDACTED, method: GET

Dec 07 10:23:42 raspberrypi java[1351]:         at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:641)

Dec 07 10:23:42 raspberrypi java[1351]:         at java.base/java.nio.HeapByteBuffer.get(HeapByteBuffer.java:162)

Dec 07 10:23:42 raspberrypi java[1351]:         at xyz.gianlu.librespot.core.Session$Receiver.run(Session.java:766)

Dec 07 10:23:42 raspberrypi java[1351]:         at java.base/java.lang.Thread.run(Thread.java:834)

Dec 07 10:25:47 raspberrypi java[1351]: 2019-12-07 10:25:47 WARN  Session:745 - Socket timed out. Reconnecting...

```

**Comment by designandcopy** [comment]
Appears to be related to #141

**Comment by devgianlu** [comment]
The issue has already been solved in version `1.0.0`, looks like `spocon` isn't updated. Alternatively you can compile it yourself, download one of the releases or use [ansible-role-librespot](https://github.com/xMordax/ansible-role-librespot).
